docs/detections/rules-ui-create.asciidoc

```diff
@@ -274,15 +274,13 @@ To preview a rule:
 +
 . Select a timeframe of data to preview query results -- *Last hour*, *Last day*, or *Last month* -- from the *Quick query preview* drop-down.
 +
-. Click *Preview results*. A histogram shows the number of alerts you can expect based on the defined rule parameters and historical events in your indices.
-
-A "noise warning" is displayed if the preview generates more than one alert per hour.
-
+. Click *Preview results*. The rule preview shows a histogram and alerts table with the alerts you can expect, based on the defined rule parameters and historical events in your indices. You can view the details of a particular alert by clicking the *View details* button in the alerts table.
++
+NOTE: The preview excludes the effects of rule exceptions and timestamp overrides. In the preview histogram, alerts are stacked by `event.category` (or `host.name` for machine learning rules), and events with multiple values are counted more than once.
```

Comment:
@dplumlee Does this better represent how the preview works? I also updated the screenshot to include multiple categories to illustrate the stacking.

Reply:
Yep this looks great! Thanks!